main/inc/lib/hook/HookAdminBlock.class.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
```

Comment:
You forgot the license terms.
You left the PHP Storm header.
Please configure your PHPStorm to start a new file with the right headers:
<?php
 /* For licensing terms, see /license.txt */
/**
* Script explanation
* @package chamilo.
*/

So you only have to fill it.